src/services/catalog.service.ts

```diff
@@ -439,7 +439,7 @@ export class ImageItem implements IServiceItem {
   }
 
   private getVendor(): string {
-    return _.get(this.resource, 'metadata.providerDisplayName') as string || '';
+    return '';
```

Comment:
Should we just remove the vendor field from the ImageItem class?

Reply:
It's in the `IServiceItem` interface as a public field, although I could remove the method and just initialize it to `''`

Reply:
I see. It's OK as is then. Maybe someday it has one.

Reply:
@bparees Any opinion on adding the `template.openshift.io/provider-display-name` annotation to builder images? Without it, all of the builders will disappear in the UI when you filter for Provider: Red Hat

Reply:
you mean to the imagestreamtags?

Reply:
i'd prefer we come up with an annotation that's not tied to templates, for setting on things that aren't templates.

Reply:
you're also going to have to get all the xpaas imagestreamtags updated w/ whatever annotation.

Reply:
@bparees The image stream shows up as one item in the catalog, so it'd probably need to be on the image stream itself rather than the image stream tags if we want the filter to work.

Agree about the annotation name.

Reply:
> you're also going to have to get all the xpaas imagestreamtags updated w/ whatever annotation.

@bparees Yeah, @sspeiche and I were just talking about the xpaas image streams. I think we could get the annotation added.

Reply:
I'm open to adding it.